mobileapp/components/register-login/SignIn.tsx

```diff
@@ -0,0 +1,203 @@
+import {
+  StyleSheet,
+  Text,
+  SafeAreaView,
+  View,
+  Modal,
+  KeyboardAvoidingView,
+  Platform,
+  Pressable,
+  Image,
+} from "react-native";
+import * as React from "react";
+import { TextInput, TouchableOpacity } from "react-native-gesture-handler";
+import { useSelector } from "react-redux";
+import * as Actions from "../../redux/actions";
+import * as Utility from "../../redux/utility.js";
+import * as styles from "../../constants/account";
+import { State } from "../../redux/reducer";
+import { store } from "../../redux/store";
+import { clearLoginInformation } from "../../redux/actions";
+
+const SignIn = ({ navigation }: any) => {
+  const [username, setUsername] = React.useState("");
```

Comment:
So like it seems the linter is a little bit broken, but add types to use state
Like, `useState<string>("");`, it should not break the app

Reply:
Done